modules/datagrid/launch/added/launch/custom-logging.sh

```diff
@@ -18,8 +18,13 @@ function configure() {
           loggers+="\n            <logger category=\"$category\"><level name=\"$level\"/></logger>"
           loggercount=$((loggercount+1))
         done
-        sed -i "s|<!-- ##ADDITIONAL_LOGGER## -->|$loggers|" "$CONFIG_FILE"
       fi
+  else
+    # then use default loggers
```

Comment:
@ryanemerson this solution seems most robust to me. Here we define current "default" loggers and if no value is specified for CUSTOM_LOGGERS we output these three. Otherwise, we go with the custom.